gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterManager.java

```diff
@@ -135,6 +135,8 @@
   private GobblinHelixJobScheduler jobScheduler;
   @Getter
   private JobConfigurationManager jobConfigurationManager;
+  @Getter
+  private volatile boolean started = false;
```

Comment:
So how is this flag going to be used? I see that it is only set to true, but never read in OSS codebase.

Reply:
The use of this flag is similar to the use of [this](https://github.com/apache/gobblin/blob/master/gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinTaskRunner.java#L163) flag. Right now it will be used by linkedin internal code.